src/jira/client/jira-client-deployment-helper.ts

```diff
@@ -16,3 +18,21 @@ export const getDeploymentDebugInfo = (jiraPayload: JiraDeploymentBulkSubmitData
 		commitCount: associations.filter(a => a.associationType === "commit").map(a => a.values?.length || 0).reduce((a, b) => a + b, 0)
 	};
 };
+
```

Comment:
This code is as was, pulled from jiraclient file and moved here.... Tests are new though